tests/trainer/test_trainer_seq2seq.py

```diff
@@ -161,7 +161,6 @@ def test_return_sequences(self):
             tokenizer=tokenizer,
             data_collator=data_collator,
             compute_metrics=lambda x: {"samples": x[0].shape[0]},
-            report_to="none",
```

Comment:
is this intended?

Reply:
Yes, the argument does not exist and error introduced from here: https://github.com/huggingface/transformers/pull/30266

Reply:
ok thanks !

Reply:
Done!